makedev.ps1

```diff
@@ -2,4 +2,12 @@ $REPO_PATH = "github.com/Azure/aks-engine"
 $DEV_ENV_IMAGE = "quay.io/deis/go-dev:v1.23.2"
 $DEV_ENV_WORK_DIR = "/go/src/$REPO_PATH"
 
+# Ensure docker is configured for linux containers
+$json = (docker version --format "{{json .}}" | ConvertFrom-Json)

```

Comment:
or `docker version --format "{{json .Server.Os }}"` either way works

Reply:
Thank, i'll keep that in mind for the future.